drake/systems/plants/@Manipulator/parameterEstimation.m

```diff
@@ -34,17 +34,26 @@
 % @option model determines which model to use for estimation
 % 'dynamic'     = use dynamic model - requires qdd
 % 'energetic'   = use energetic model - doesn't require qdd
+% 'simerr'      = minimize simulation error
```

Comment:
write out simulation error?  make the distinction clear with a few more words here.